docker-compose.yml

```diff
@@ -153,6 +153,8 @@ services:
       - "logging=true"
     restart: always
     logging: *default-logging
+    volumes:
+      - ./data/files/burgemeester-benoemingen:/uploads
```

Comment:
It should just work! But when uploading a file it returns file uploaded but it is not put in the folder

Reply:
@JonasVanHoof that is very strange. Please check because this is exactly what this pr is supposed to fix. Possibly drc kill and rm first.
Once done, can you merge and deploy this?